src/Microsoft.DotNet.Build.Tasks.Feed/BlobUrlInfo.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Auth;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    class BlobUrlInfo
```

Comment:
nit: Based on our coding guidelines we generally make the visibility explicitly listed.

Reply:
What coding guidelines are in use in htis repo? Is it tied to corefx?

Reply:
Yes we generally use the corefx guidelines.